jena-shex/src/main/java/org/apache/jena/shex/parser/javacc/ShExJavacc.java

```diff
@@ -1,4 +1,3 @@
-/* ShExJavacc.java */
```

Comment:
Again unclear how much of the diff here is due to using an older JavaCC version vs actual functional changes from the grammar changes